### ISSUE TITLE
Color scheme GUI is light instead of dark

### ISSUE THREAD

**Issue by patmu** [locaal-ai/scoresight#50]
On my Windows 11 computer, the software gui is almost white instead of the darker colors in the tutorial videos.

![image](https://github.com/user-attachments/assets/016fc500-c0d2-4c88-b540-054017d2cb45)

Is there a switch to change the color schema to dark? With the white one it's very difficult to work.

Best, Patrick

**Comment by royshil** [comment]
this is not determined by scoresight rather by the operating system. i didn't have to set anything and i get it dark

**Comment by patmu** [comment]
You're right: it's the OS color mode in Windows 11 which is responsible for the ScoreSight color scheme.

![image](https://github.com/user-attachments/assets/20b0f268-6b4b-48a5-85e2-971d66111c89)

I suggest to set the colors or the scheme in QT itself because in light mode the individual controls are almost not visible. Or to make it switchable in the application itself and ignore the OS value.

**Comment by royshil** [comment]
i could add a way to override theme and set it up in the settings

**Comment by patmu** [comment]
Thanks for considering.
